_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,176 @@
+- The notwithin_coordinates_factory() and density_from_PDB() methods of analysis.density are deprecated.
+- analysis.waterdynamics.HydrogenBondLifetimes is deprecated in favor of analysis.hydrogenbonds.hbond_analysis.HydrogenBondAnalysis.lifetime() (to be implemented in version 2.0.0)
+- analysis.leaflets.LeafletFinder() will no longer accept a filename, in 2.0.0 only Universes will be supported as inputs.
+- analysis.helanal is deprecated and will be replaced by analysis.helix_analysis in 2.0.0.
+- analysis.hbonds.WaterBridgeAnalysis will be moved to analysis.hydrogenbonds.WaterBridgeAnalysis.
+
+
+The following parts of the readers/writers will be removed/changed in version 2.0.0:
+- Writer.write_next_timestep() is deprecated in favor of Writer.write().
+- Passing Timestep objects to Writer.write() is deprecated. In 2.0.0 only Universe or AtomGroup objects will be accepted.
+- The way in which the NCDFWriter handles scale factos will change in version 2.x (see Issue #2327 for more details).
+- When writing PDB files, MDAnalysis will no longer be using the last letter of the SegID to set the chainID in version 2.0.0.
+- The bfactors and tempfactors attributes (set by the PDB and MMTF parsers respectively), will be aliased in version 2.0.0.
+- When parsing TPR files, resids will be indexed from 1 rather than the current default of 0.
+
+The following part of the core and library components will be removed/changed in version 2.0.0:
+- lib.log.echo() is deprecated in favor of the new lig.log.ProgressBar.
+- core.universe.as_Universe() is deprecated.
+
+## Author statistics
+
+Altogether this represents the work of 42 contributors from around the world,
+and featured the work of 25 new contributors:
+
+ - [@bdice](https://github.com/bdice)
+ - [@joaomcteixeira](https://github.com/joaomcteixeira)
+ - [@CCook96](https://github.com/CCook96)
+ - [@Yuan-Yu](https://github.com/Yuan-Yu)
+ - [@HTian1997](https://github.com/HTian1997)
+ - [@Iv-Hristov](https://github.com/Iv-Hristov)
+ - [@hmacdope](https://github.com/hmacdope)
+ - [@AnshulAngaria](https://github.com/AnshulAngaria)
+ - [@ss62171](https://github.com/ss62171)
+ - [@Luthaf](https://github.com/Luthaf)
+ - [@yuxuanzhuang](https://github.com/yuxuanzhuang)
+ - [@abhishandy](https://github.com/abhishandy)
+ - [@mlnance](https://github.com/mlnance)
+ - [@shfrz](https://github.com/shfrz)
+ - [@wvandertoorn](https://github.com/wvandertoorn)
+ - [@cbouy](https://github.com/cbouy)
+ - [@AmeyaHarmalkar](https://github.com/AmeyaHarmalkar)
+ - [@Oscuro-Phoenix](https://github.com/Oscuro-Phoenix)
+ - [@andrrizzi](https://github.com/andrrizzi)
+ - [@WG150](https://github.com/WG150)
+ - [@Marcello-Sega](https://github.com/Marcello-Sega)
+ - [@calcraven](https://github.com/calcraven)
+ - [@ianmkenney](https://github.com/ianmkenney)
+ - [@rcrehuet](https://github.com/rcrehuet)
+ - [@hanatok](https://github.com/hanatok)
```

Comment:
Someone did a lot of extra work here — you can just write @orbeckst and it will automatically link to the GH profile; this is part of the GH packages that parse the markdown.